src/pages/terms-of-service/index.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useEffect, useState } from "react";
+
+const TermsOfService = () => {
+  const [isChecked, setChecked] = useState(false);
+  const [showScrollButton, setShowScrollButton] = useState(false);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      const contentElement = document.querySelector(".content");
+      const isScrolledToBottom =
+        contentElement.scrollHeight - contentElement.scrollTop - 1 >=
+        contentElement.clientHeight;
+
+      setShowScrollButton(isScrolledToBottom);
+    };
+
+    const contentElement = document.querySelector(".content");
+    contentElement.addEventListener("scroll", handleScroll);
```

Comment:
ini contentElement perlu di kasi null safety. Coba cari cara di chat gpt aja mas. Make sure element not null